WordPress/build.gradle

```diff
@@ -102,6 +103,7 @@ android {
             buildConfigField "boolean", "MODAL_LAYOUT_PICKER", "false"
             buildConfigField "boolean", "WP_STORIES_AVAILABLE", "false"
             buildConfigField "boolean", "ANY_FILE_UPLOAD", "false"
+            buildConfigField "boolean", "ENABLE_FEATURE_CONFIGURATION", "false"
```

Comment:
Since the description of zalpha is: 
```
zalpha { // alpha version - enable experimental features
```
I thought it fits in there 🤷 , I'm fine with disabling it. Who gets zalpha?

Reply:
Alpha is used for closed testing track distributed through Google Play. However, since the purpose of this screen is to allow devs+testers test features which are not prepared for a release yet I believe we don't need to have this screen in internal testing phase. We might want to consider disabling it there for now and enable it if we find the need. Wdyt?

Reply:
makes sense. I'll disable it here and leave it on only for wasabi and jalapeno